Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -1059,3 +1049,21 @@ private func visitorCookies() -> [HTTPCookie] {
   )
   .compact()
 }
+
+private func qualtricsConfigData() -> QualtricsConfigData {
+  return .init(
+    brandId: Secrets.Qualtrics.brandId,
+    zoneId: Secrets.Qualtrics.zoneId,
+    interceptId: QualtricsIntercept.survey.interceptId,
+    stringProperties: [
+      "bundle_id": AppEnvironment.current.mainBundle.bundleIdentifier,
+      "language": AppEnvironment.current.language.rawValue,
+      "logged_in": "\(AppEnvironment.current.currentUser != nil)",
+      "distinct_id": AppEnvironment.current.device.identifierForVendor?.uuidString,
+      "user_uid": AppEnvironment.current.currentUser.flatMap { $0.id }.map(String.init),
+      "hours_since_joined": AppEnvironment.current.currentUser?
+        .hoursSinceJoined(AppEnvironment.current.dateType.init().date).flatMap(String.init)
```

Comment:
does `String.init` mean you're sending all these properties as strings? should `hoursSinceJoined` be a number?

Reply:
resolving this since we no longer have this property